### ISSUE TITLE
bleno and eddyston-beacon don't play nice together

### ISSUE THREAD

**Issue by beaufortfrancois** [don/node-eddystone-beacon#30]
I'm currently experiencing some issues with an Edison-like board and I was wondering if you could help me there.
I'm basically using https://github.com/sandeepmistry/bleno and https://github.com/don/node-eddystone-beacon libraries and I'm having a hard time making these work together.

My use case is simple, basic:
- Advertise only eddystone-URL
- If someone connects to my device, stop advertising & populate services
- Let user interact with my services/characteristics

Would you, by any chance, have some sample code I could use as a basis?

@sandeepmistry @don

**Comment by don** [comment]
Try  https://gist.github.com/don/01b41297554378235519

It works better on OS X 10.10. On Ubuntu 14.04, it looks like the beacon info clobbers the service advertisement. (Which might be OK if you're using the beacon for service discovery?)

**Comment by sandeepmistry** [comment]
@don's gist works well on my Raspberry Pi as well. However, if I use `npm install bleno eddystone-beacon` instead of `npm install` with the `package.json`, I think i get two instances of bleno, as nRF Master control panel shows 2 0x1800 and 0x1801 services, and no echo service.

**Comment by sandeepmistry** [comment]
To clarify on my comment above, I actually ran:

```
npm install eddystone-beacon
npm install bleno
```

so bleno was install twice is the directory tree ...

**Comment by beaufortfrancois** [comment]
See my comment at https://github.com/don/node-eddystone-beacon/pull/31#issuecomment-161218939

**Comment by beaufortfrancois** [comment]
We can close this one then.
Thank you for your help!

**Comment by WayneKeenan** [comment]
The connectable.js example no longer works when using bleno 0.4.0   

It works ok using bleno 0.3.5 on the same system (RaspberryPi 3 with Jessie)

I added the dependency explicitly in the example package.json as I would like to use this on RPi and v0.4.0 has a necessary fix for the Pi 3's BLE h/w.

[bleno0.3.5.hcidump.txt](https://github.com/don/node-eddystone-beacon/files/189736/bleno0.3.5.hcidump.txt)
[bleno0.4.0.hcidump.txt](https://github.com/don/node-eddystone-beacon/files/189737/bleno0.4.0.hcidump.txt)

**Comment by sandeepmistry** [comment]
@WayneKeenan please submit a pull request to update the bleno dependency to 0.4.0 then either myself or @don can publish a new version to npm.

**Comment by PaulB2Code** [comment]
Hi @sandeepmistry , 

Is there any news on this subjects. I tried on edison but i can't get working bleno and eddystone together. 

Each working separatly but together it is like eddystone works but not bleno as the device can not be scanned. 

I try the full path method, the connectable Js example, the gist example (https://gist.github.com/don/01b41297554378235519)  but none are working.

Notice (For the front I used only webbluetooth) 

Thank you for your help.
